Vagrantfile

```diff
@@ -3,19 +3,41 @@ url             = 'https://install.codenvycorp.com/centos7.2.box'
 
 ram             = '4096'
 cpus            = '2'
-bridge          = 'eth0'
+#bridge          = 'eth0'
+
+# Set to "<proto>://<user>:<pass>@<host>:<port>"
+http_proxy      = ENV['HTTP_PROXY'] || ""
+https_proxy     = ENV['HTTPS_PROXY'] || ""
+no_proxy        = ENV['NO_PROXY'] || "localhost,127.0.0.1"
```

Comment:
It may be reasonable to add codenvy host url to the list of no-proxy dns, that is:
`codenvy.onprem,localhost,127.0.0.1`